accounts-bench/Cargo.toml

```diff
@@ -10,10 +10,10 @@ homepage = "https://solana.com/"
 [dependencies]
 log = "0.4.6"
 rayon = "1.4.0"
-solana-logger = { path = "../logger", version = "1.4.0" }
-solana-runtime = { path = "../runtime", version = "1.4.0" }
-solana-measure = { path = "../measure", version = "1.4.0" }
-solana-sdk = { path = "../sdk", version = "1.4.0" }
+solana-logger = { path = "../logger", version = "1.3.6" }
+solana-runtime = { path = "../runtime", version = "1.3.6" }
+solana-measure = { path = "../measure", version = "1.3.6" }
+solana-sdk = { path = "../sdk", version = "1.3.6" }
```

Comment:
@carllin what's with these version changes?

Reply:
fffff I think it must've crept in from another PR, good catch!